src/http/routing.rs

```diff
@@ -27,23 +27,24 @@ enum RatelimitingKind {
 macro_rules! routes {
     ($lt:lifetime, {
         $(
-            $name:ident $({ $($field_name:ident: $field_type:ty),* })?,
+            $name:ident $({ $($field_name_route:ident: $field_type:ty | $field_type_owned:ty),* })?,
+            { $($field_name:ident),* },
             $path:expr,
             $ratelimiting_kind:expr;
         )+
     }) => {
+        #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+        enum RouteKind {
+            $($name,)+
+        }
```

Comment:
Why was this moved?

Reply:
To have two similar blocks next to another (`Route` and `OwnedRoute`).
I could move it back, but in my opinion it is more readable like this.